From 8104d603977b4b4ef33e9c613a0b7c2b8aa762c7 Mon Sep 17 00:00:00 2001
From: Torge Matthies <openglfreak@googlemail.com>
Date: Mon, 25 Apr 2022 05:24:26 +0200
Subject: [PATCH 07/16] Revert "user32: Add support for absolute rawinput
 messages."

This reverts staging patch patches/user32-rawinput-mouse-experimental/0002-user32-Add-support-for-absolute-rawinput-messages.patch
---
 dlls/win32u/rawinput.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/dlls/win32u/rawinput.c b/dlls/win32u/rawinput.c
index 11111111111..11111111111 100644
--- a/dlls/win32u/rawinput.c
+++ b/dlls/win32u/rawinput.c
@@ -96,8 +96,7 @@ static bool rawinput_from_hardware_message( RAWINPUT *rawinput, const struct har
         rawinput->header.hDevice = WINE_MOUSE_HANDLE;
         rawinput->header.wParam  = 0;
 
-        rawinput->data.mouse.usFlags = msg_data->flags & MOUSEEVENTF_ABSOLUTE ? MOUSE_MOVE_ABSOLUTE : MOUSE_MOVE_RELATIVE;
-        if (msg_data->flags & MOUSEEVENTF_VIRTUALDESK) rawinput->data.mouse.usFlags |= MOUSE_VIRTUAL_DESKTOP;
+        rawinput->data.mouse.usFlags           = MOUSE_MOVE_RELATIVE;
         rawinput->data.mouse.usButtonFlags = 0;
         rawinput->data.mouse.usButtonData  = 0;
         for (i = 1; i < ARRAY_SIZE(button_flags); ++i)
-- 
2.37.1

